frontend/packages/process-editor/src/components/ConfigPanel/EditDataType/EditDataType.test.tsx

```diff
@@ -120,12 +157,20 @@ describe('EditDataType', () => {
   });
 });
 
-const renderEditDataType = (bpmnApiContextProps: Partial<BpmnApiContextProps> = {}) => {
+type RenderProps = {
+  bpmnApiContextProps: Partial<BpmnApiContextProps>;
+  rootContextProps: Partial<BpmnContextProps>;
+  componentProps: Partial<EditDataTypeProps>;
```

Comment:
Might be a personal preference, but I dont see the need for adding props that are not used in the test 🫣 `rootContextProps` in this case